### PR TITLE
data: Rohit312001

### PR DESCRIPTION
**Pull request by Rohit312001**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/7572"></a>

### CODE REVIEW

File: data/Rohit312001.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "Rohit Rajput",
+  "bio": "DevOps Enthusiast | GSSoC'23 | SSoC'23 | UIET Chandigarh.I do Development and Technical blogging on Hashnode.I really enjoy learning languages and DevOps tools",
+   "links": [
+    {
+      "name": "Rohit Rajput",
+      "url": "https://github.com/Rohit312001",
+      "icon": "FaGithub"
+    }
```
Comment (SaraJaoude) [inline data/Rohit312001.json:9]: add a comma:
```suggestion
    },
```

File: data/Rohit312001.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    {
+      "name": "Rohit Rajput",
+      "url": "https://www.linkedin.com/in/rohit-rajput-68779a202/",
+      "icon": "FaLinkedin"
+    }
```
Comment (SaraJaoude) [inline data/Rohit312001.json:14]: add a comma:
```suggestion
    },
```

File: data/Rohit312001.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+      {
+      "name": "Rohit Rajput",
+      "url": "https://hashnode.com/@rohit8329",
+      "icon": "SiHashnode"
+    }
```
Comment (SaraJaoude) [inline data/Rohit312001.json:19]: add a comma:
```suggestion
    },
```

**Review by SaraJaoude — APPROVED** [review]
looks good

**Review by Rohit312001 — COMMENTED** [review]
Thanks

**Pull request merged**